src/components/seo/validations.js

```diff
@@ -0,0 +1,17 @@
+export const validateDescription = value => {
+  const maxLength = 180
+
+  if (!value) {
+    return null
+  }
+
+  if (!(typeof value == "string" || value instanceof String)) {
```

Comment:
Do we really need the second condition? As far as I'm aware the first one is enough for strings. Also, shouldn't the first one be using strict comparison?

```suggestion
  if (typeof value !== "string") {
```

Reply:
string primitives and string objects are two different things in JS. Unless there's some `isString()` function that encapsulates this nowadays, this is the recommended way to do this check

Reply:
the non-strict was an oversight. fixing